addons/web/controllers/main.py

```diff
@@ -588,6 +588,29 @@ def translations(self, mods=None, lang=None):
         return {"modules": translations_per_module,
                 "lang_parameters": lang_params}
 
+    @http.route('/web/webclient/technical_translations', type='json', auth="user")
+    def technical_translations(self, model):
+        fields = request.env['ir.model.fields'].search([('model', '=', model)])
+        trans = request.env['ir.translation'].search([
+            '|',
+                '&',('type', '=', 'model'),
+                    '&',('res_id', 'in', fields.ids),
+                        '|',
+                            ('name', '=', 'ir.model.fields,field_description'),
+                            ('name', '=', 'ir.model.fields,help'),
+                '&',('type', '=', 'selection'),
+                    ('name', 'ilike', model),
+        ])
+        return {
+            'name': _("Technical Translation"),
+            'view_mode': 'tre,form',
```

Comment:
typo: `'tree,form'`
BTW you should not need the form view here, isn't the list view editable?